chatrooms-rules.md

```diff
@@ -28,7 +28,11 @@ These are currently only enforced on **Anonymity** and **Security**, and not app
 - No NSFW content (no Porn, no Gore, no Hentai...)
 - No upload of any non-media files (binaries, executables, compressed files...)
 - No voice messages (these will be auto-deleted by the bots)
-- **Avoid drifting too much off-topic or move to an off-topic room like #bnonymity**
+- External bots are forbidden unless explicit permission from an Admin
+- Moderation for small offenses (off-topic talking for example) will follow a 3 step system: Warning > Kick > Ban (those can be appealed,  see the end of this page).
+- Moderation for heavy offenses (CSAM for example) might result in an immediate ban and possibly an immediate permaban where an appeal will be denied. Some of those heavy offenses might result in a report to the server admins or even the authorities.
+- All kicks and bans shall have a clear reason and bans shall have an expiration time. It is up to the person being banned to contact the moderators or adminds of the room to request unban based on expiration.

```

Comment:
Double space on LN32 as well: "appealed,  see".
Typo here for adminds (sic) and LN34. 
"request unban based on expiration" -> "appeal their ban" maybe.